server/jetty/src/main/java/io/deephaven/server/jetty/ControlledCacheResource.java

```diff
@@ -31,13 +28,13 @@ private ControlledCacheResource(Resource wrapped) {
     }
 
     @Override
-    public boolean isContainedIn(Resource r) throws MalformedURLException {
-        return wrapped.isContainedIn(r);
+    public Path getPath() {
+        return wrapped.getPath();
     }
 
     @Override
-    public void close() {
-        wrapped.close();
```

Comment:
Seems odd to me that `Resource` doesn't have a `close` method anymore? Doesn't even seem to be mentioned as being removed in the [migration guide](https://jetty.org/docs/jetty/12/programming-guide/migration/11-to-12.html#api-changes). Weird.

Reply:
Yeah, they seem to have entirely rewritten the type hierarchy of Resource, anything that used to need a close() now is handled as a Path directly, and it is up to the caller of Resource.getPath() to handle IO (or one of the convenience methods on Resource, which either clean up after themselves, or return a Closeable).